### ISSUE TITLE
Abort trap 6

### ISSUE THREAD

**Issue by jlamb85** [DragonCherry/AssetsPickerViewController#109]
When archiving a Project using AssetsPickerViewController (2.9.7) I get the following error.

I am using Swift 13.3.  It works on 13.2.1

The following build commands failed: CompileSwiftSources normal armv7 com.apple.xcode.tools.swift.compiler (in target 'AssetsPickerViewController' from project 'Pods') CompileSwiftSources normal arm64 com.apple.xcode.tools.swift.compiler (in target 'AssetsPickerViewController' from project 'Pods') SwiftCodeGeneration normal arm64 (in target 'AssetsPickerViewController' from project 'Pods') (3 failures)

**Comment by vinayak-nickelfox** [comment]
Same

**Comment by ronitvasoya** [comment]
Same issue I have faced

**Comment by dimitarspasovski-movic** [comment]
do you have workaround for this issue?

**Comment by jlamb85** [comment]
Yes download Xcode 13.2.1.  That seems to be the only way right now
